Proposals/xgov-208.md

```diff
@@ -0,0 +1,94 @@
+---
+id: 208
+period: 3
+title: DigiLeaf Electric Vehicle Chargers
+author: Trenton Austin (@bigbulb)
+email: TODO
```

Comment:
Please add an Email address to start the contract process

Reply:
Updated